src/bridge/mod.rs

```diff
@@ -59,6 +59,13 @@ pub async fn show_intro_message(
     nvim.exec_lua(INTRO_MESSAGE_LUA, args).await
 }
 
+pub async fn show_error_message(
+    nvim: &Neovim<NeovimWriter>,
+    message: &String,
+) -> Result<(), Box<CallError>> {
+    nvim.command(&format!("echomsg \"{message:?}\"")).await
```

Comment:
The code might be cleaner if you use the api `nvim_echo`. Or perhaps even an error `nvim_err_writeln`. 

So, `nvim.echo` or `nvim.err_writeln` instead of `nvim.command`

Reply:
aah makes sense, thank you!